<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDataService</class>
 <widget class="QWidget" name="StudentDataService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>521</width>
     <height>253</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonAddStudent">
      <property name="styleSheet">
       <string notr="true">  QPushButton {
        background-color: #3A6EA5;
        color: #E0E6F2;
        border: none;
        border-radius: 8px;
        padding: 8px 16px;
        font-weight: 600;
    }
    QPushButton:hover {
        background-color: #5591D3;
    }
    QPushButton:pressed {
        background-color: #2C5588;
    }</string>
      </property>
      <property name="text">
       <string>Добавить Студента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSortStud">
      <property name="styleSheet">
       <string notr="true">  QPushButton {
        background-color: #3A6EA5;
        color: #E0E6F2;
        border: none;
        border-radius: 8px;
        padding: 8px 16px;
        font-weight: 600;
    }
    QPushButton:hover {
        background-color: #5591D3;
    }
    QPushButton:pressed {
        background-color: #2C5588;
    }</string>
      </property>
      <property name="text">
       <string>Отсортировать студентов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSeeFile">
      <property name="styleSheet">
       <string notr="true">  QPushButton {
        background-color: #3A6EA5;
        color: #E0E6F2;
        border: none;
        border-radius: 8px;
        padding: 8px 16px;
        font-weight: 600;
    }
    QPushButton:hover {
        background-color: #5591D3;
    }
    QPushButton:pressed {
        background-color: #2C5588;
    }</string>
      </property>
      <property name="text">
       <string>Смотреть файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDeleteStudent">
      <property name="styleSheet">
       <string notr="true">  QPushButton {
        background-color: #3A6EA5;
        color: #E0E6F2;
        border: none;
        border-radius: 8px;
        padding: 8px 16px;
        font-weight: 600;
    }
    QPushButton:hover {
        background-color: #5591D3;
    }
    QPushButton:pressed {
        background-color: #2C5588;
    }</string>
      </property>
      <property name="text">
       <string>Удалить Cтудента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonFindStudent">
      <property name="styleSheet">
       <string notr="true">  QPushButton {
        background-color: #3A6EA5;
        color: #E0E6F2;
        border: none;
        border-radius: 8px;
        padding: 8px 16px;
        font-weight: 600;
    }
    QPushButton:hover {
        background-color: #5591D3;
    }
    QPushButton:pressed {
        background-color: #2C5588;
    }</string>
      </property>
      <property name="text">
       <string>Найти Студента</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonBackMain">
      <property name="styleSheet">
       <string notr="true">  QPushButton {
        background-color: #3A6EA5;
        color: #E0E6F2;
        border: none;
        border-radius: 8px;
        padding: 8px 16px;
        font-weight: 600;
    }
    QPushButton:hover {
        background-color: #5591D3;
    }
    QPushButton:pressed {
        background-color: #2C5588;
    }</string>
      </property>
      <property name="text">
       <string>Главное Меню</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
